<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Main Feed</string>
    </property>
    <widget class="QWidget" name="AppFeed" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>445</width>
       <height>445</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start video</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>1470</x>
      <y>10</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>461</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Alternate</string>
    </property>
    <widget class="QWidget" name="AppFeedAlt" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>445</width>
       <height>445</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ToggleAltButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>560</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle Alternate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ProblemDescription">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>100</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Problem Description</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="InputGroupBox">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>400</y>
      <width>491</width>
      <height>281</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="ImageAlgorithm1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>convert color</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ImageAlgorithm2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>126</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>find in range</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ImageAlgorithm3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>152</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>erosion</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ImageAlgorithm3_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>178</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>dialation</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ImgAlgSubmitButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TextSubmitButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>30</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ImgAlgResetButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CheckBoxSubmitButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CheckBoxResetButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>100</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RadialBoxResetButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>240</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RadialSubmitButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="ConfidenceBar">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>16</width>
       <height>160</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Your confidence level for this problem.</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QScrollBar" name="DifficultyBar">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>16</width>
       <height>160</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>the difficulty of this problem</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="FormSubmitButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>240</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Liberation Serif</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Submit All</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>60</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>confidence</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>46</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>difficulty</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>434</x>
       <y>200</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="HColorSlider">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>590</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>H color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSlider" name="SColorSlider">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>630</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>S Color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSlider" name="VColorSlider">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>670</y>
      <width>160</width>
      <height>18</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>V Color for HSV colorspace</string>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksBelow</enum>
    </property>
    <property name="tickInterval">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>590</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>H</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>630</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>670</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Liberation Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>60</y>
      <width>571</width>
      <height>661</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tab 3</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>580</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(-5,-3)</string>
    </property>
   </widget>
   <widget class="QLabel" name="directionLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>580</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QLabel" name="distanceTravelledLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>580</y>
      <width>67</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>URW Bookman L</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Location          Direction    Spaces Moved</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="robotTextBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>421</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>630</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Robot Output</string>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>groupBox</zorder>
   <zorder>startButton</zorder>
   <zorder>progressBar</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>ToggleAltButton</zorder>
   <zorder>ProblemDescription</zorder>
   <zorder>InputGroupBox</zorder>
   <zorder>HColorSlider</zorder>
   <zorder>SColorSlider</zorder>
   <zorder>VColorSlider</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>locationLabel</zorder>
   <zorder>directionLabel</zorder>
   <zorder>distanceTravelledLabel</zorder>
   <zorder>label_8</zorder>
   <zorder>robotTextBox</zorder>
   <zorder>label_9</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
